src/App.tsx

```
@@ -5,7 +5,7 @@ import {
   QueryClient,
   QueryClientProvider,
 } from '@tanstack/react-query';
-import { useState } from 'react';
+import { useState, useEffect } from 'react';
```

ESLint

[error] 'useEffect' is defined but never used.

(@typescript-eslint/no-unused-vars)